.env.example

```diff
@@ -1,7 +1,9 @@
 # Secrets
 NEXT_PUBLIC_SUPABASE_URL=http://127.0.0.1:54321
+NEXT_PUBLIC_SUPABASE_SERVICE_KEY =
```

Comment:
It's not necessary to add this env variable. With the `NEXT_PUBLIC_SUPABASE_ANON_KEY` it's more than enough to access Supabase